libs/blocks/preflight/panels/accessibility.js

```diff
@@ -0,0 +1,58 @@
+import { html, signal, useEffect } from '../../../deps/htm-preact.js';
+
+const DEF_ICON = 'purple';
+const DEF_DESC = 'Checking...';
+const pass = 'green';
+const fail = 'red';
+
+const content = signal({});
+const altResult = signal({ icon: DEF_ICON, title: 'Image alt value', description: DEF_DESC });
+
+async function checkAlt() {
+  const main = document.querySelector('main');
+  const images = main.querySelectorAll('img');
+  const result = { ...altResult.value };
+  const imagesWithoutAlt = [];
+  images.forEach((img) => {
+    const alt = img.getAttribute('alt');
+    if (!alt || alt.trim() === '') {
+      imagesWithoutAlt.push(img.getAttribute('src'));
+    }
+  });
+  if (!imagesWithoutAlt.length) {
+    result.icon = pass;
+    result.description = 'Reason: All Image are valid';
```

Comment:
Nit: image should be plural and image should not be capitalized.
```suggestion
    result.description = 'Reason: All images are valid.';
```

Reply:
Fixed